src/standalone/Snow/Snow.jl

```diff
@@ -332,6 +332,36 @@ function SnowParameters{FT}(
     )
 end
 
+## For interfacing with ClimaParams
+
+SnowParameters(::Type{FT}, Δt; kwargs...) where {FT <: AbstractFloat} =
```

Comment:
Can you add a docstring for this like you have for the canopy param constructors?